src/util/ColorsUtil.js

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright (c) 2018-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(function () {
+  var fn = {};
+
+  var lighten = function (hex, lum) {
+    hex = String(hex).replace(/[^0-9a-f]/gi, '');
+    var newHex = '#';
+    for (var i = 0; i < 3; i++) {
+      var c = parseInt(hex.substr(i * 2, 2), 16);
+      c = Math.round(Math.min(Math.max(0, c + (c * lum)), 255)).toString(16);
+      newHex += ('00' + c).substr(c.length);
+    }
+    return newHex;
+  };
+
+  var template = function (colors) {
+    return `
+      #okta-sign-in.auth-container .button-primary,
```

Comment:
Use of multi-line template literals oftentimes causes unexpected indentations.

Can we bring the indentation to the far left, so the return object will be:

```css
#okta-sign-in.auth-container .button-primary,
#okta-sign-in.auth-container .button-primary:active,
#okta-sign-in.auth-container .button-primary:focus { background: abc123; }
#okta-sign-in.auth-container .button-primary:hover { background: 0.05; }
#okta-sign-in.auth-container .button.button-primary.link-button-disabled {
  background: abc123;
  opacity: 0.5;
}
```

Instead of: 
```css
      #okta-sign-in.auth-container .button-primary,
      #okta-sign-in.auth-container .button-primary:active,
      #okta-sign-in.auth-container .button-primary:focus { background: abc123; }
      #okta-sign-in.auth-container .button-primary:hover { background: 0.05; }
      #okta-sign-in.auth-container .button.button-primary.link-button-disabled {
        background: abc123;
        opacity: 0.5;
      }
```